_includes/saved_searches.md

```diff
@@ -0,0 +1,62 @@
+## Save and Manage Searches
+
+Because narrowing down a search in the Application Status page or Traces Browser can be time consuming, you can
+* Save a search. Saved searches are available to all members of your organization.
+* Access and delete your saved searches
+* Access saved searches from everyone in your organization.
+* Set a saved search as the default search.
+
+### Save and Access a Search
+
+After you've performed a search on the Application Status page or Traces Browser, you can save your search.
+
+1. Put in the search parameters, such as applications or services, add filters, and click <strong>Search</strong>.
+1. Click the bookmark icon, specify a name for your search, and click <strong>Save</strong>.
+   ![Bookmark icon brings up save search dialog](/images/save_a_search_button.png)
+1. Click the <strong>Lists</strong> button to the left of the query.
+   ![List button](/images/saved_searches_list_button.png)
+1. In the dialog that appears: 
+    <ul>
+      <li>Click <strong>My Recent Searches</strong> for a list of your recent searches. </li>
+      <li>Click <strong>My Saved Searches</strong> for a complete list of your searches. </li>
+      <li>Click <strong>All Saved Searches</strong> for a list of searches in your organization.  </li>
+    </ul>
+    ![List button](/images/saved_searches_list.png)
+    
+1. Click a saved search to select it.
+
+### Delete Saved Searches
+
+The Lists button gives you access to your own saved searches and to saved searches from others in your organization.
```

Comment:
Button names are always bold. I'd reword this to:  By clicking the **Lists** button, you can access your own saved searches and the saved searches from others in your organization.